idemp-rest-test/testing/src/tests/inventory.test.ts

```diff
@@ -0,0 +1,17 @@
+import { patientApi, referenceListApi, visitApi, productApi } from '../api';
+import { referenceUuid, tenderTypeName } from '../models';
+import { Payment, PaymentType, Visit } from '../types/org.bandahealth.idempiere.rest';
+import { createPatient, createProduct, createVisit, waitForVisitToComplete, createProductInventory } from '../utils';
+
+test(`Product Created with negative value`, async () => {
+	const valueObject = globalThis.__VALUE_OBJECT__;
+	await valueObject.login();
+
+	valueObject.stepName = 'Create product with negative value';
```

Comment:
This test will need to create a BP before creating a product, then receive the product, then sell more than received afterwards. Can you update?

Reply:
Updating